environs/config/config.go

```diff
@@ -2055,7 +2055,7 @@ global or per instance security groups.`,
 	},
 	ProvisionerHarvestModeKey: {
 		// default: destroyed, but also depends on current setting of ProvisionerSafeModeKey
-		Description: "What to do with unknown machines. See https://jujucharms.com/stable/config-general#juju-lifecycle-and-harvesting (default destroyed)",
```

Comment:
we *should* have docs on how to set these sorts of things, but certainly that one doesn't exist (that I can find).